storybook/src/components/Mark/Mark.stories.tsx

```diff
@@ -3,14 +3,14 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { Mark, Paragraph } from '@amsterdam/design-system-react'
+import { Mark, Paragraph } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
 
 const meta = {
   title: 'Components/Text/Mark',
   component: Mark,
   args: {
-    children: 'Nieuw component',
+    children: 'Wat vinden Amsterdammers belangrijk?',
```

Comment:
Not sure whether this should be the default value if we only have one story. Chances are that a second story would mark a different phrase and we’d move this text back again. I’d say move it back and remove the default arg.